app/assets/javascripts/Components/markdown_edit.jsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import MarkdownPreview from "./markdown_preview";
+import {Tab, Tabs, TabList, TabPanel} from "react-tabs";
+
+import PropTypes from "prop-types";
+
+export default class MarkdownEdit extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      content: this.props.content,
+      annotation_text_id: "",
+    };
+  }
+
+  handleChange = event => {
+    this.setState({content: event.target.value});
+    this.props.handleChange(event);
+  };
+
+  render() {
+    return (
+      <Tabs disableUpDownKeys>
+        <TabList>
+          <Tab id="write-tab">{I18n.t("write")}</Tab>
```

Comment:
I don't think setting `id` on React components (instead of HTML elements) has any effect, though maybe I could be wrong. But in any case if you aren't using these ids you should remove them.

Reply:
The id is set on the second tab for testing. I set it on the first tab as well for consistency. I will remove it form the first tag.